include/tune_quda.h

```diff
@@ -243,7 +243,33 @@ namespace quda {
     virtual void apply(const qudaStream_t &stream) = 0;
     virtual void preTune() { }
     virtual void postTune() { }
-    virtual int tuningIter() const { return 1; }
+
+    /**
+     * @brief Parameter to control the number of iteration used in the 2nd phase of tuning, i.e. for the candidates.
+     *
+     * @return int minimum number of iterations
+     */
+    virtual int min_tune_iter() const { return 3; }
```

Comment:
I think the order of these declarations should be reversed, such that the parameters for the first stage of tuning should come first, and the candidate (2nd) tuning should come second.